src/transformers/file_utils.py

```diff
@@ -213,6 +214,7 @@
 PYTORCH_PRETRAINED_BERT_CACHE = os.getenv("PYTORCH_PRETRAINED_BERT_CACHE", default_cache_path)
 PYTORCH_TRANSFORMERS_CACHE = os.getenv("PYTORCH_TRANSFORMERS_CACHE", PYTORCH_PRETRAINED_BERT_CACHE)
 TRANSFORMERS_CACHE = os.getenv("TRANSFORMERS_CACHE", PYTORCH_TRANSFORMERS_CACHE)
+TRANSFORMERS_ID = uuid4().hex
```

Comment:
I had a preference for the previous name `SESSION_ID` that I find more informative.